### DISCUSSION TITLE
数据相关

### DISCUSSION THREAD

**Discussion by sakura-umi** [SakuraLLM/SakuraLLM#53] [General]
数据集相关问题讨论区

**Comment by daiaji** [comment]
请问数据集在哪里？我想试着用来微调gemini。

**Reply by sakura-umi** [reply]
训练sakura所用的数据集并不公开。

该区主要讨论训练数据格式和公开数据集的相关问题。

**Reply by CberYellowstone** [reply]
> 训练sakura所用的数据集并不公开。
> 
> 该区主要讨论训练数据格式和公开数据集的相关问题。

不公开数据集的话，能公开数据集列表吗，想要自己整理，但是缺乏方向

**Reply by sakura-umi** [reply]
包括有繁体中文出版的轻小说与有官方简体中文翻译的galgame
